src/strategies/api-post/examples.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "api-post",
+      "params": {
+        "api": "https://dao-staging.polychainmonsters.com/v1/votingPowerTestMock",
+        "symbol": "Points",
+        "strategy": "PointsByAddress",
```

Comment:
This param doesnt seem to be used, could you remove it?

Reply:
@bonustrack 
done ✅